elvis.config

```diff
@@ -3,14 +3,28 @@
   elvis,
   [
    {config,
-    [#{dirs => ["apps/*/src", "apps/*/test", "test/ejabberd_tests/tests"],
+    [#{dirs => ["apps/*/src"],
        filter => "*.erl",
        ruleset => erl_files,
        rules => [{elvis_style, line_length, #{limit => 100,
                                               skip_comments => false}},
 		 {elvis_style, dont_repeat_yourself, #{min_complexity => 20}}
 		]
-      }
+      },
+     #{dirs => ["apps/*/test", "test/ejabberd_tests/tests"],
+       filter => "*.erl",
+       ruleset => erl_files,
+       rules => [{elvis_style, line_length, #{limit => 100,
+                                              skip_comments => false}},
+                 {elvis_style, dont_repeat_yourself, disable},
+                 {elvis_style, invalid_dynamic_call, disable},
+                 {elvis_style, no_debug_call, disable},
+                 {elvis_style, god_modules, disable},
+                 {elvis_style, no_spec_with_records, disable},
+                 {elvis_style, used_ignored_variable, disable},
```

Comment:
Thats not really possible. I found just one error that correspond to this check and it's usage of macro "_assert_equal_extra". Eagain - thats a naming convention in those macros. On the other hand I don't really understand why Elvis treats macros like variables. 
Sooo, either we remove this check in tests (It's critical no to use ignored variables in tests) or we have to accept the fact that Elvis may never be fully happy with our tests. 

For me, the second option is worse, because if we say that is fine to merge not happy PR's (because some checks have to fail), there is a chance that we stop checking elvis in tests at all.